--- origsrc/libxslt-1.1.24/configure.in	2008-05-13 10:40:31.000000000 -0500
+++ src/libxslt-1.1.24/configure.in	2008-07-09 02:20:10.921875000 -0500
@@ -510,6 +510,13 @@
 WIN32_EXTRA_LIBADD=
 WIN32_EXTRA_LDFLAGS=
 case "$host" in
+ *-*-cygwin*)
+ WIN32_EXTRA_LDFLAGS="-no-undefined"
+ if test "${PYTHON}" != ""
+ then
+   CYGWIN_EXTRA_PYTHON_LIBADD="-L/usr/lib/python${PYTHON_VERSION}/config -lpython${PYTHON_VERSION}"
+ fi
+ ;;
  *-*-mingw*)
  WIN32_EXTRA_LIBADD="-lwsock32"
  WIN32_EXTRA_LDFLAGS="-no-undefined"
@@ -520,6 +527,7 @@
 esac
 AC_SUBST(WIN32_EXTRA_LIBADD)
 AC_SUBST(WIN32_EXTRA_LDFLAGS)
+AC_SUBST(CYGWIN_EXTRA_PYTHON_LIBADD)
 
 
 AC_SUBST(XSLTPROCDV)
--- origsrc/libxslt-1.1.24/libexslt/Makefile.am	2007-01-03 09:11:57.000000000 -0600
+++ src/libxslt-1.1.24/libexslt/Makefile.am	2008-07-09 02:20:10.921875000 -0500
@@ -29,7 +29,7 @@
 	dynamic.c
 
 libexslt_la_LIBADD = $(top_builddir)/libxslt/libxslt.la $(EXTRA_LIBS)
-libexslt_la_LDFLAGS = -version-info @LIBEXSLT_VERSION_INFO@ $(LIBGCRYPT_LIBS)
+libexslt_la_LDFLAGS = -version-info @LIBEXSLT_VERSION_INFO@ $(WIN32_EXTRA_LDFLAGS) $(LIBGCRYPT_LIBS)
 
 man_MANS = libexslt.3
 
--- origsrc/libxslt-1.1.24/libxslt/Makefile.am	2007-01-03 09:11:57.000000000 -0600
+++ src/libxslt-1.1.24/libxslt/Makefile.am	2008-07-09 02:20:10.937500000 -0500
@@ -51,7 +51,7 @@
 	libxslt.h
 
 libxslt_la_LIBADD = $(EXTRA_LIBS)
-libxslt_la_LDFLAGS = -version-info @LIBXSLT_VERSION_INFO@
+libxslt_la_LDFLAGS = -version-info @LIBXSLT_VERSION_INFO@ $(WIN32_EXTRA_LDFLAGS)
 
 man_MANS = libxslt.3
 
--- origsrc/libxslt-1.1.24/python/Makefile.am	2007-01-03 09:11:42.000000000 -0600
+++ src/libxslt-1.1.24/python/Makefile.am	2008-07-09 02:20:10.937500000 -0500
@@ -24,12 +24,14 @@
 	libxslt-python-api.xml	\
 	$(DOCS)
 
-libxsltmod_la_LDFLAGS = -module -avoid-version
+libxsltmod_la_LDFLAGS = -module -avoid-version $(WIN32_EXTRA_LDFLAGS)
 
 if WITH_PYTHON
 mylibs = \
 	$(top_builddir)/libxslt/libxslt.la \
-	$(top_builddir)/libexslt/libexslt.la
+	$(top_builddir)/libexslt/libexslt.la \
+	$(CYGWIN_EXTRA_PYTHON_LIBADD) \
+	$(EXTRA_LIBS)
 
 all-local: libxslt.py
 
