--- origsrc/libxslt-1.1.26/configure.in	2009-09-24 09:27:30.000000000 -0500
+++ src/libxslt-1.1.26/configure.in	2009-12-19 23:18:57.134830100 -0600
@@ -625,6 +625,10 @@ WIN32_EXTRA_LDFLAGS=
 case "$host" in
  *-*-cygwin*)
  WIN32_EXTRA_LDFLAGS="-no-undefined"
+ if test -n "${PYTHON}"
+ then
+   CYGWIN_EXTRA_PYTHON_LIBADD="-L/usr/lib/python${PYTHON_VERSION}/config -lpython${PYTHON_VERSION}"
+ fi
  ;;
  *-*-mingw*)
  WIN32_EXTRA_LIBADD="-lwsock32"
@@ -636,6 +640,7 @@ case "$host" in
 esac
 AC_SUBST(WIN32_EXTRA_LIBADD)
 AC_SUBST(WIN32_EXTRA_LDFLAGS)
+AC_SUBST(CYGWIN_EXTRA_PYTHON_LIBADD)
 
 
 AC_SUBST(XSLTPROCDV)
--- origsrc/libxslt-1.1.26/python/Makefile.am	2009-12-19 22:22:05.066671000 -0600
+++ src/libxslt-1.1.26/python/Makefile.am	2009-12-19 22:30:48.333600200 -0600
@@ -29,7 +29,8 @@ libxsltmod_la_LDFLAGS =  $(WIN32_EXTRA_L
 if WITH_PYTHON
 mylibs = \
 	$(top_builddir)/libxslt/libxslt.la \
-	$(top_builddir)/libexslt/libexslt.la
+	$(top_builddir)/libexslt/libexslt.la \
+	$(CYGWIN_EXTRA_PYTHON_LIBADD)
 
 all-local: libxslt.py
 
