--- origsrc/libxslt-1.1.24/doc/Makefile.am	2007-01-15 06:54:32.000000000 -0600
+++ src/libxslt-1.1.24/doc/Makefile.am	2009-03-16 21:59:18.015625000 -0500
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 
 # The name of the module.
-DOC_MODULE=libxslt-$(VERSION)
+DOC_MODULE=libxslt
 
 # The top-level SGML file.
 DOC_MAIN_XML_FILE=libxslt.xml
--- origsrc/libxslt-1.1.24/python/Makefile.am	2009-03-16 21:53:14.375000000 -0500
+++ src/libxslt-1.1.24/python/Makefile.am	2009-03-16 22:00:24.234375000 -0500
@@ -10,7 +10,7 @@ INCLUDES = \
 	-I$(top_srcdir) \
 	-I../libexslt
 
-DOCS_DIR = $(datadir)/doc/libxslt-python-$(LIBXSLT_VERSION)
+DOCS_DIR = $(datadir)/doc/python-libxslt
 # libxsltclass.txt is generated
 DOCS = TODO
 
--- origsrc/libxslt-1.1.24/python/tests/Makefile.am	2007-01-03 09:11:42.000000000 -0600
+++ src/libxslt-1.1.24/python/tests/Makefile.am	2009-03-16 22:00:12.234375000 -0500
@@ -1,4 +1,4 @@
-EXAMPLE_DIR = $(datadir)/doc/libxslt-python-$(LIBXSLT_VERSION)/examples
+EXAMPLE_DIR = $(datadir)/doc/python-libxslt/examples
 
 TESTSPY=	\
     basic.py	\
